gsva/gsva.xml

```diff
@@ -124,38 +120,18 @@
             </when>
         </conditional>
         
-        <section name="viz" title="Data Visualizations" expanded="false">
-            <conditional name="box_plots">
-                <param name="do" type="select" label="Produce Box and Whisker Plots">
-                    <option value="do">Do</option>
-                    <option value="dont">Don't</option>
-                </param>
-                <when value="do">
-                    <param name="box_name" type="text" label="Plot Name" value="gsva_box_plot.png" />
-                    <param name="sample" type="text" label="Sample ID" help="Sample identifier, as specified by the study cohort" />
-                </when>
-                <when value="dont">
-                </when>
-            </conditional>
-            <conditional name="heatmap">
-                <param name="do" type="select" label="Produce Clustered Heatmap">
-                    <option value="do">Do</option>
-                    <option value="dont">Don't</option>
-                </param>
-                <when value="do">
-                    <param name="hm_name" type="text" label="Plot name" value="gsva_heatmap.png" />
-                </when>
-                <when value="dont">
-                </when>
-            </conditional>
+        <section name="visuals" title="Data Visualizations" expanded="false">
+            <param name="box_name" type="text" label="Plot Name" value="gsva_box_plot.png" />
+            <param name="sample" type="text" label="Sample ID" help="Sample identifier, as specified by the study cohort" />
+            <param name="hm_name" type="text" label="Plot name" value="gsva_heatmap.png" />
```

Comment:
Would prefer to not allow user to specify file name. Suggest giving files the default names you have here and control file naming at the workflow level.

Reply:
Default names implemented. Will be visible in the next push.